src/style.css

```diff
@@ -1 +1,85 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.search__input--big {
+  height: 70px;
+  padding-left: 62px;
+  background-size: 19px;
+  background-position: 26px center;
+  font-size: 16px;
+}
+
+.search__input--big:hover {
+  box-shadow: 0 4px 4px 0 #00000040;
+}
+
+.search__input--big:focus {
+  font-weight: bold;
+  box-shadow: 0 4px 4px 0 #00000040;
+  text-shadow: 0 4px 4px 0 #00000040;
+  background: url(images/Search.svg)
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
it doesn't work:

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/5890de24-a071-48cd-b2d0-bee865fb938f)

```suggestion
  background: url(images/Search.svg),
    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```
add a comma between different backgrounds. Bot you will need to adjust all image background properties additionally by using separate properties (like background-repeat).

OR add image via `img` tag with `position: absolute`